subsystems/sidecar/lib/http.js

```diff
@@ -164,7 +164,9 @@ module.exports = class Http extends ReadyResource {
         }
         app.warmup({ protocol, batch })
       }
-      const stream = await bundle.streamFrom(link.filename)
+
+      const meta = await bundle.getMeta(link.filename)
+      const stream = meta === null ? null : bundle.streamFrom(meta)
```

Comment:
I kept the behaviour exactly as-is, but I'm 99% sure there's another bug here: when meta is null, `      await streamx.pipelinePromise(stream, res)` throws. Unsure what the expected behaviour should be though.